tests/test_random_list_new_chainability.py

```
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)